modules/reference/pages/rpk/rpk-cluster/rpk-cluster-config.adoc

```diff
@@ -1,24 +1,32 @@
 = rpk cluster config
+// tag::single-source[]
 
 Interact with cluster configuration properties.
 
-Cluster properties are redpanda settings which apply to all nodes in
-the cluster. These are separate to node properties, which are set with
+ifdef::env-cloud[]
+Cluster properties are Redpanda settings that apply to all brokers in
+the cluster.
+endif::[]
+
+ifndef::env-cloud[]
+Cluster properties are Redpanda settings that apply to all brokers in
+the cluster. These are separate to broker properties, which are set with
 `rpk redpanda config`.
 
 Use the `edit` subcommand to interactively modify the cluster configuration, or
-`export` and `import` to write configuration to a file that can be edited and
+`export` and `import` to write the configuration to a file that can be edited and
 read back later.
```

Comment:
those options are not available in rpk cloud config, only set and status for now

Reply:
@andresaristizabal Ah yes, the `ifndef` will ensure that this part of the description does not render in the Cloud version of this doc